specs/rider_spec.rb

```diff
@@ -0,0 +1,62 @@
+require_relative 'spec_helper'
+
+describe RideShare::Rider do
+    let(:new_rider) { RideShare::Rider.new(256) }
+
+    # rider-info: 256,Kiara Kuhn,458.159.0504 x540
+
+    describe 'initialize' do
+        it 'Creates a new instance of Rider' do
+            new_rider.must_be_instance_of RideShare::Rider
+        end
+        it 'Initializes with the correct information' do
+            new_rider.rider_id.must_equal 256
+            new_rider.name.must_equal 'Kiara Kuhn'
+            new_rider.phone_number.must_equal '458.159.0504 x540'
+        end
+        it 'Raises an error if incorrect id is entered' do
+            proc { RideShare::Rider.new('wrong') }.must_raise ArgumentError
+            proc { RideShare::Rider.new('$*2hus') }.must_raise ArgumentError
+            proc { RideShare::Rider.new(-47) }.must_raise ArgumentError
+        end
+        it 'Raises an error if ID doesnt exist' do
+            proc { RideShare::Rider.new(8_387_764) }.must_raise ArgumentError
+            proc { RideShare::Rider.new(0) }.must_raise ArgumentError
+        end
+    end
+    describe 'previous_drivers' do
+        it 'Returns an array of accurate driver data' do
+            new_rider.previous_drivers[0].driver_id.must_equal 69
+            new_rider.previous_drivers[0].name.must_equal 'Ernesto Torp'
+            new_rider.previous_drivers[0].vin.must_equal 'RF4BPA803R4AACTR1'
+        end
```

Comment:
As with `Driver.average_rating`, what happens if the rider hasn't taken any trips?